### PR TITLE
Add more linters and address related issues

### PR DESCRIPTION
**Pull request by hezhizhen**

### CODE REVIEW

**Comment by hezhizhen** [comment]
Close https://github.com/wakatime/wakatime-cli/pull/834 with correct branch name

**Comment by hezhizhen** [comment]
I need some help to pass unit tests on Windows @alanhamlett @gandarez

**Comment by alanhamlett** [comment]
I re-ran that test and it passed the second time. On Windows, sometimes the GitHub Action fails to install the linter and we just re-run it.

**Comment by hezhizhen** [comment]
> I re-ran that test and it passed the second time. On Windows, sometimes the GitHub Action fails to install the linter and we just re-run it.

Thanks.

**Comment by alanhamlett** [comment]
LGTM, but I'll wait for @gandarez to make the final decision before merging.

File: .golangci.yml [hunk 1]
```diff
@@ -15,10 +15,15 @@ linters:
     - gofmt
     - goimports
     - gosec
+    - gosimple
     - misspell
     - nestif
     - revive
     - testpackage
+    - unconvert
+    - unparam
```
Comment (gandarez) [inline .golangci.yml]: Unused params is already set using `revive:unused-parameter`

File: .golangci.yml [hunk 2]
```diff
@@ -15,10 +15,15 @@ linters:
     - gofmt
     - goimports
     - gosec
+    - gosimple
```
Comment (gandarez) [inline .golangci.yml]: It's enabled by default, so don't need to explicitly set here.

File: .golangci.yml [hunk 3]
```diff
@@ -15,10 +15,15 @@ linters:
...
     - misspell
     - nestif
     - revive
     - testpackage
+    - unconvert
+    - unparam
+    - unused
```
Comment (gandarez) [inline .golangci.yml]: It's enabled by default, so don't need to explicitly set here.

**Review by gandarez — CHANGES_REQUESTED** [review]

**Comment by hezhizhen** [comment]
I removed the 3 linters. @gandarez

**Review by gandarez — APPROVED** [review]

**Pull request merged**